tapestry.js

```diff
@@ -862,6 +907,111 @@ function setupMedia(id, mediaFormat, mediaType, mediaUrl, width, height) {
     return mediaEl;
 }
 
+// Builds the view mode, including functionality to
+function changeToViewMode(adjustedOn) {
+    // For calculating where the node should go
+    var nodeRatio = getBrowserWidth() / getTapestryDimensions().width;
+    var nodeRadius = NORMAL_RADIUS * nodeRatio;
+    var nodeSpace = (nodeRadius * 2) + ((nodeRadius * 2) * 0.10);
+
+    var children = getChildren(root);
+    var coordinates = [];
+    for (var i = 0; i < children.length; i++) {
+        if (adjustedOn === "width") {
+            if (i % 2 === 0) {
+                coordinates[children[i]] = {
+                    "fx": 0,
+                    "fy": Math.min(getTapestryDimensions().height * (i / (children.length - 1)) + nodeRadius, getTapestryDimensions().height - nodeRadius)
+
+            };
+            } else {
+                coordinates[children[i]] = {
+                    "fx": getTapestryDimensions().width - nodeSpace,
```

Comment:
Again, I think you'll need to add NODE_RADIUS here otherwise the node will be half hidden behind the player